eng/AfterSolutionBuild.targets

```diff
@@ -0,0 +1,31 @@
+<Project>
```

Comment:
Hmm I'm not sure if this is being used at all but it's probably important since @natemcmaster added it explicitly.

Reply:
Yes, this used. This file is imported by Arcade and invoked as a part of build.cmd/sh